lib/matplotlib/backends/backend_agg.py

```diff
@@ -124,7 +124,7 @@ def draw_path(self, gc, path, transform, rgbFace=None):
         nmax = rcParams['agg.path.chunksize']  # here at least for testing
         npts = path.vertices.shape[0]
 
-        if (nmax > 100 and npts > nmax and path.should_simplify and
+        if (100 < nmax < npts and path.should_simplify and
```

Comment:
I think this reads more naturally as `npts > nmax > 100` (`npts` is reallt the interesting variable here).